org.gnomeradio.Radio.appdata.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <application>
-  <id type="desktop">org.gnomeradio.Radio</id>
```

Comment:
why did you remove this?

Reply:
Invalid tag?

Reply:
Fixed.

Reply:
![image](https://github.com/flathub/flathub/assets/114441/340eb127-d649-4a20-90a6-96476978ea1a)


Not sure what part is unclear.